packages/edit-post/src/style.scss

```diff
@@ -99,6 +99,10 @@ body.block-editor-page {
 
 @include wordpress-admin-schemes();
 
+.interface-interface-skeleton__content {
```

Comment:
Instead of adding this override, could we see if we can rework [the original `overscroll: auto` prop in the `interface` package](https://github.com/WordPress/gutenberg/blob/0ab7645ef7c5b37f0f80d43ec35c60ed488d7937/packages/interface/src/components/interface-skeleton/style.scss#L85)?

Reply:
If I'm not mistaken the site editor now uses an iframe, meaning the overflow auto here isn't particularly useful since it's the iframe that now handles the scroll

Reply:
Which also means we can't set it to scroll (it would result in a double scrollbar)

Reply:
Got it!

What I'm trying to say is, we couldn't set those styles for `.interface-interface-skeleton__content` and `.interface-interface-skeleton__header` (at the moment `scrollbar-gutten` and `overflow`) directly at the source (in the `@wordpress/interface` package), instead of overriding them specifically for the `@wordpress/edit-post` package.